setup.sh

```diff
@@ -138,6 +160,14 @@ if [ "$upgrading" = 1 ]; then
 	# Get current var path
 	var_dir=`cat $config_dir/var-path`
 
+	# Get current bootscript
+	if [ -r "$config_dir/bootscript-name" ]; then
+		bootscript=`cat $config_dir/bootscript-name`
+		if [ "$bootscript" = "" ]; then
```

Comment:
Also this should be moved outside the if statement

Reply:
All of these variables are not block-scoped. These are visible globally!

Reply:
But what if the `bootscript-name` file doesn't exist? `$bootscript` will never be set to `webmin`

Reply:
Oh, I see where confusion is coming from. It will. Please check [line 12](https://github.com/webmin/webmin/pull/1715/files/dcdf0c59ae16449e48d06acf8831fd1338d0cf46#diff-4209d788ad32c40cbda3c66b3de47eefb929308ca703bb77a6382625986add17R12-R15) on this PR.

Reply:
Oh I see now ... yeah, that's better.

Reply:
> Oh I see now ... yeah, that's better.

It was already there, I didn't just add it. 🙂